docs/docs/unit-testing.md

```diff
@@ -242,13 +242,13 @@ Run the tests again now and it should all work! You should get a message about
 the snapshot being written. This is created in a `__snapshots__` directory next
 to your tests. If you take a look at it, you will see that it is a JSON
 representation of the `<Bio />` component. You should check your snapshot files
-into a source control system (for example, a GitHub repo) so that so that any changes are tracked in history.
+into a source control system (for example, a GitHub repo) so that any changes are tracked in history.
 This is particularly important to remember if you are using a continuous
 integration system such as Travis to run tests, as these will fail if no
 snapshot is present.
 
 If you make changes that mean you need to update the snapshot, you can do this
-by running `npm run test -- -u`.
+by running `npm run test --updateSnapshot -u`.
```

Comment:
-u is the alias for --updateSnapshot. Also while yarn is supporting this syntax `yarn test --updateSnapshot` npm does not. You have to write it like `npm run test -- --updateSnapshot` this to pass down the updateSnapshot flag to the command. So this should be either:
`npm run test -- --updateSnapshot`
or
`npm run test -- -u`

Reply:
Thank you. I don't know about `--` option. I will close this pull request